tests/installation/bootloader_svirt.pm

```diff
@@ -56,6 +57,31 @@ sub search_image_on_svirt_host {
     return $path;
 }
 
+# wrapper for vmware shell cmd execution
+# res: 0 passed, 1 failed. Does 'assertion' when errmsg provided
+sub check_run_command {
```

Comment:
```suggestion
sub run_command {
```
I don't think the check adds much meaning to the function name here.

Reply:
renamed